acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/NodeConfiguration.java

```diff
@@ -22,11 +22,20 @@
   private final String hostname;
   private final String genesisFilePath;
   private final Optional<String> cors;
+  private final boolean privacyEnabled;
+  private final Optional<String> privacyPublicKeyPath;
 
-  public NodeConfiguration(final String genesisFilePath, final String hostname, final String cors) {
+  public NodeConfiguration(
+      final String genesisFilePath,
+      final String hostname,
+      final String cors,
+      final boolean privacyEnabled,
```

Comment:
can this just be if the privacy key is provided? - i.e. AT does not need an invalid configuration of Pantheon to test EthSigner.

Reply:
no I don't think we need to test that